static/app/views/traces/fieldRenderers.spec.tsx

```diff
@@ -0,0 +1,315 @@
+  SpanIdRenderer,
+  TraceIdRenderer,
+  TraceIssuesRenderer,
+  TransactionRenderer,
+} from 'sentry/views/traces/fieldRenderers';
+
+describe('Renderers', function () {
+  let context;
+
+  const organization = OrganizationFixture({
+    features: ['trace-view-v1'], // only testing against new trace view
+  });
+
+  const projects = [
+    ProjectFixture({
+      id: '1',
+      slug: 'project-1',
+      name: 'Project 1',
+    }),
+    ProjectFixture({
+      id: '2',
+      slug: 'project-2',
+      name: 'Project 2',
+    }),
+    ProjectFixture({
+      id: '3',
+      slug: 'project-3',
+      name: 'Project 3',
+    }),
+  ];
+
+  function makeSpan(
+    project: Project,
+    span?: Partial<SpanResult<Field>>
+  ): SpanResult<Field> {
+    return {
+      project: project.slug,
+      'transaction.id': '00000000000000000000000000000000',
+      id: '11111111111111111111111111111111',
+      timestamp: '2024-07-03T10:15:00',
+      'sdk.name': 'sentry.python',
+      'span.op': 'op',
+      'span.description': 'description',
+      'span.duration': 123.456,
+      'span.status': 'internal_error',
+      'span.self_time': 12.34,
+      'precise.start_ts': 1720015976.544,
+      'precise.finish_ts': 1720016100.0,
+      is_transaction: 1,
+      ...span,
+    };
+  }
+
+  beforeEach(function () {
+    context = initializeOrg({organization, projects});
+    act(() => ProjectsStore.loadInitialData(projects));
+
+    MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/projects/`,
+      body: projects,
+    });
+  });
+
+  describe('SpanDescriptionRenderer', function () {
+    it('renders op then description', function () {
+      const span = makeSpan(projects[0]);
+
+      render(<SpanDescriptionRenderer span={span} />);
+
+      const description = screen.getByTestId('span-description');
+      expect(description).toBeInTheDocument();
+      expect(description).toHaveTextContent('op\u2014descriptioninternal_error');
+    });
+
+    it.each(['unknown', 'foobar'])(
+      'does not render span status %s',
+      function (spanStatus) {
+        const span = makeSpan(projects[0], {'span.status': spanStatus});
+
+        render(<SpanDescriptionRenderer span={span} />);
+
+        const description = screen.getByTestId('span-description');
+        expect(description).toBeInTheDocument();
+        expect(description).toHaveTextContent('op\u2014description');
+      }
+    );
+
+    it.each([
+      'ok',
+      'cancelled',
+      'invalid_argument',
+      'deadline_exceeded',
+      'not_found',
+      'already_exists',
+      'permission_denied',
+      'resource_exhausted',
+      'failed_precondition',
+      'aborted',
+      'out_of_range',
+      'unimplemented',
+      'internal_error',
+      'unavailable',
+      'data_loss',
+      'unauthenticated',
+    ])('renders span status %s', function (spanStatus) {
+      const span = makeSpan(projects[0], {'span.status': spanStatus});
```

Comment:
Unless you have a strong reason to suspect other values would not render or other error paths, I'd just do `ok` and maybe one other since this list is just testing the backend sent data. Otherwise we're just running tests for mocks that will always be true.


```suggestion
      'ok',
    ])('renders span status %s', function (spanStatus) {
      const span = makeSpan(projects[0], {'span.status': spanStatus});
```